components/AppBar/README.md

```diff
@@ -1,12 +1,13 @@
 <!--docs:
-title: "App Bar"
+title: "App Bars"
```

Comment:
That said, we have this one:
> Buttons (p): Floating Action Button (s)

Think it's singular because of usage, or because they were running out of space? :)